unifi/Dockerfile

```diff
@@ -1,21 +1,39 @@
-ARG BUILD_FROM=ghcr.io/hassio-addons/ubuntu-base:8.2.0
+ARG BUILD_FROM=ghcr.io/hassio-addons/ubuntu-base:9.0.3
 # hadolint ignore=DL3006
 FROM ${BUILD_FROM}
 
+ARG BUILD_ARCH=amd64
+
 # Set shell
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 # Setup base system
+
+# mongodb-org-server is installed from the community servers as it's gone from
+# repositories for newer ubuntu versions.
+#
+# Beware of the licence change, previously the addon used mongodb version 3.6.9
+# released under the AGPL, while 4.0.3 and newer are released under the non-free SSPL
+# https://en.wikipedia.org/wiki/MongoDB#Licensing
 RUN \
     apt-get update \
     && apt-get upgrade -y \
     && apt-get install -y --no-install-recommends \
-        binutils=2.34-6ubuntu1.8 \
-        libcap2=1:2.32-1ubuntu0.1 \
-        logrotate=3.14.0-4ubuntu3 \
-        mongodb-server=1:3.6.9+really3.6.8+90~g8e540c0b6d-0ubuntu5.3 \
+        binutils=2.38-4ubuntu2.6 \
+        logrotate=3.19.0-1ubuntu1.1 \
         openjdk-17-jre-headless=17* \
     \
+    && pkgname=libssl1.1_1.1.1f-1ubuntu2.21 \
+    && curl -L -O "http://security.ubuntu.com/ubuntu/pool/main/o/openssl/${pkgname}_amd64.deb" \
+    && curl -L -O "http://ports.ubuntu.com/pool/main/o/openssl/${pkgname}_arm64.deb" \
```

Comment:
Let's make sure we only fetch the one we need, instead of both.

Reply:
Fixed